### PR TITLE
Double star dictionary

### PR DESCRIPTION
**Pull request by Jollokim**
Implementation of addparkingplace in ListRepository, where it has multiple params, but can still take a dictionary as param as long as the ** is put in front of it and keys are accordingly. Also moved a placeholder method in Controller to Repository.

### CODE REVIEW

**Comment by Jollokim** [comment]
Can also make new parkingplaces, and they will show and operate correct in the "Leier section"

File: parkx_kode/model/Parkingplace.py [hunk 1]
```diff
@@ -2,17 +2,18 @@
...
-        self.number_of_places = number_of_places
-        self.price_pr_hour = price_pr_hour
+        self.number_of_places = int(number_of_places)
+        self.price_pr_hour = int(price_pr_hour)
         self.picture = picture
         self.details = details
-        self.available = available
-        self.parkingStarted = datetime.datetime.now().strftime("%H:%M:%S")
+        self.available = True
+        # self.parkingStarted = datetime.datetime.now().strftime("%H:%M:%S")
```
Comment (ThomasJohannessen) [inline parkx_kode/model/Parkingplace.py]: skal denne være kommentert ut?

File: parkx_kode/repository/ListRepository.py [hunk 2]
```diff
@@ -5,19 +5,15 @@ class ListRepository:
...
-            p_dict["available"] = True
-
-        new_pp = Parkingplace(p_dict["id"], p_dict["name"], p_dict["address"],
-                              p_dict["zip_code"], p_dict["number_of_places"],
-                              p_dict["price_pr_hour"], p_dict["picture"],
-                              p_dict["details"], p_dict["available"])
+    def addNewParkingPlace(self, id, name, address, zip_code, number_of_places, price_pr_hour, picture, details):
+        new_pp = Parkingplace(id, name, address, zip_code, number_of_places, price_pr_hour, picture, details)
 
+        print(new_pp.toString())
```
Comment (ThomasJohannessen) [inline parkx_kode/repository/ListRepository.py]: skal denne print være her?

File: parkx_kode/repository/ListRepository.py [hunk 3]
```diff
@@ -28,6 +24,7 @@ def getPP(self, id):
             if pp.id == id:
                 return pp
 
+    # TODO: needs double start for p_dict
```
Comment (ThomasJohannessen) [inline parkx_kode/repository/ListRepository.py:26]: skal denne inn i kanban?

File: parkx_kode/view/gui.py [hunk 4]
```diff
@@ -22,14 +22,23 @@ class Gui(BoxLayout):
...
+    ]
 
     def __init__(self, controller,  **kwargs):
         super(Gui, self).__init__(**kwargs)
 
         self.controller = controller
 
         self.PPs_list = controller.get_all_pp_from_list()
-        self.controller.addPlaceholderPlaces()
+        # self.controller.addPlaceholderPlaces()
```
Comment (ThomasJohannessen) [inline parkx_kode/view/gui.py]: kommentert ut?

File: parkx_kode/model/Parkingplace.py (hunk 1 above)
Comment (Jollokim) [reply parkx_kode/model/Parkingplace.py]: skal fjerne den fullstendig. takk for du ser over

File: parkx_kode/repository/ListRepository.py (hunk 2 above)
Comment (Jollokim) [reply parkx_kode/repository/ListRepository.py]: fjerner

File: parkx_kode/repository/ListRepository.py (hunk 3 above)
Comment (Jollokim) [reply parkx_kode/repository/ListRepository.py:26]: den skal kanskje det egentlig ja

File: parkx_kode/view/gui.py (hunk 4 above)
Comment (Jollokim) [reply parkx_kode/view/gui.py]: skal fjerne

**Review by ThomasJohannessen — APPROVED** [review]

**Pull request merged**